Peer Dependencies are incorrect for ids-enterprise-ng 11.1

**Describe the bug**

The peerdependencies are incorrect for ids-enterprise-ng 11.1

```
 "peerDependencies": {
    "@angular/common": "^13.10.0",
    "@angular/core": "^13.10.0"
  },
```

Both should be `13.1.0`.

**To Reproduce**

This should be easily reproducible in the quickstart by attempting to upgrade to 11.1.0-beta.0 and running 

```powershell
npx @angular/cli@13 update @angular/core@13 @angular/cli@13
```

(or possibly npm i)

**Expected behavior**

It should be possible to install ids-enterprise-ng when angular 13.1 is also installed.

**Version**
- ids-enterprise-ng: 11.1.0-beta.0

**Screenshots**

![image](https://user-images.githubusercontent.com/8340092/145695698-316fbc8b-ee1d-47bc-aa93-dd46384fd2f3.png)

**Platform**

N/A

**Additional context**

N/A

```powershell C:\git\github\infor-design\enterprise-ng-quickstart>npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: ids-enterprise-ng-quickstart@10.11.0
npm ERR! Found: @angular/common@13.0.3
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~13.0.3" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^13.10.0" from ids-enterprise-ng@11.1.0-beta.0
npm ERR! node_modules/ids-enterprise-ng
npm ERR!   ids-enterprise-ng@"^11.1.0-beta.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\tharper\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\tharper\AppData\Local\npm-cache\_logs\2021-12-12T10_57_06_431Z-debug.log

C:\git\github\infor-design\enterprise-ng-quickstart>
```

--- a/projects/ids-enterprise-ng/package.json
+++ b/projects/ids-enterprise-ng/package.json
@@ -12,8 +12,8 @@
     "tslib": "^2.0.0"
   },
   "peerDependencies": {
-    "@angular/common": "^13.10.0",
-    "@angular/core": "^13.10.0"
+    "@angular/common": "^13.1.0",
+    "@angular/core": "^13.1.0"
   },
   "description": "Infor Design System (IDS) Enterprise Angular Components",
   "repository": {

